code/modules/reagents/reagent_containers/glass/beaker.dm

```diff
@@ -110,23 +110,27 @@
 	desc = "A small plastic vial."
 	icon_state = "vial_plastic"
 	matter = list(MATERIAL_PLASTIC = 1)
-
+	spawn_blacklisted = TRUE
 
 /obj/item/weapon/reagent_containers/glass/beaker/vial/vape/berry
 	name = "berry vape vial"
 	preloaded_reagents = list("nicotine" = 20, "berryjuice" = 10)
+	spawn_blacklisted = TRUE
 
 /obj/item/weapon/reagent_containers/glass/beaker/vial/vape/lemon
 	name = "lemon vape vial"
 	preloaded_reagents = list("nicotine" = 20, "lemonjuice" = 10)
+	spawn_blacklisted = TRUE
 
 /obj/item/weapon/reagent_containers/glass/beaker/vial/vape/banana
 	name= "banana vape vial"
 	preloaded_reagents = list("nicotine" = 20, "banana" = 10)
+	spawn_blacklisted = TRUE
 
 /obj/item/weapon/reagent_containers/glass/beaker/vial/vape/nicotine
 	name = "nicotine vape vial"
 	preloaded_reagents = list("nicotine" = 30)
+	spawn_blacklisted = TRUE
```

Comment:
They already have it when you put blacklist in /obj/item/weapon/reagent_containers/glass/beaker/vial/vape

Reply:
Uhhh. Are you sure about this one big T...? Because there are other subtypes with blacklisted supertypes. Like /obj/rig_module/vision is marked as a bad path to prevent the parent item from spawning, but /obj/rig_module/vision/thermal and others still spawn.

Or does the spawn_blacklisted var work differently? I haven't even looked to be honest. I'll double check before making the requested changes.

Reply:
Whoops, looks like spawn_blacklisted does blacklist subtypes. Fixed.

Reply:
All these items are from the parent /obj/item/weapon/reagent_containers/glass/beaker/vial/vape of course they will get that blacklist already